src/testSimulator/Student.java

```diff
@@ -0,0 +1,63 @@
+package testSimulator;
+
+public class Student {
+
+	private int annualGrade;
+	private Answer answer;
```

Comment:
That field is never used. Do you use notepad for coding? Or simply don't give a shit about editor's warnings?